parsl/channels/base.py

```diff
@@ -20,8 +20,8 @@ class Channel(metaclass=ABCMeta):
     such as push_file, pull_file and makedirs.
 
     Channels should ensure that each launched command runs in a new process
-    group, so that providers (such as AdHocProvider and LocalProvider) which
-    terminate long running commands using process groups can do so.
+    group, so that providers (such LocalProvider) which terminate long
+    running commands using process groups can do so.
```

Comment:
"such as"?  Or "i.e.," since it's now the only one?

Reply:
there was an 'as' there before and now there is again.